.gitignore

```diff
@@ -5,3 +5,4 @@
 _build/
 dist/
 venv/
+*.patch
```

Comment:
This change is not related to the bug you are working on. Please remove it. Maybe you want to add it to your local global .gitignore file.